bin/oneoff/fix_level_group_data

```diff
@@ -44,71 +44,65 @@ DATA_FILE = '/tmp/user_level_ids.csv'.freeze
 # if raw_data is not of the appropriate format.
 # @param raw_data [Hash] The LevelGroup value (within LevelSource.data) for a
 #   sublevel.
-# @return [String] The string to persist in LevelSource.data for the sublevel.
+# @return [JSON] The JSON to persist in LevelSource.data for the sublevel.
 def get_data(raw_data)
   unless raw_data.keys == ['result', 'valid']
     raise "raw_data malformed: #{raw_data}"
   end
 
-  return raw_data['result']
+  return raw_data['result'].to_json
```

Comment:
Leaving it for now, I may end up reverting this particular change, depending on how I decide to handle some edge cases.